bin/hermit.hcl

```diff
@@ -5,6 +5,7 @@ env = {
 
   // Python:
   "LANG": "en_US.UTF-8",
+  "TZ": "UTC",
```

Comment:
nit: Could we set it in the devcontainer rather than in the Hermit env? I'd rather not change it locally (which is where I'm using Hermit from) and possibly chase a bug or have this weird unexplained inconsistency at some point :sweat_smile: